openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -214,8 +214,10 @@ func (e *extension) fetchLoggingHost(ctx context.Context) string {
 	return route.Status.Ingress[0].Host
 }
 
+// cleanupOldResources util function to clean up old, deprecated or dangling resources from Serving features.
 func (e *extension) cleanupOldResources(ctx context.Context, ns string) error {
 	client := e.kubeclient
+	// DomainMapping related resources
```

Comment:
s/DomainMapping/old resources

Reply:
I've actually added closing DomainMapping section at L254 in case we will need to split the functions to have a bit of reference.

Reply:
I see, there are leases related to net-certmanager. Added another comment and removed "closing" DomainMapping. It should be a bit more clearer now.